packages/fury-adapter-oas3-parser/lib/parser/oas/parseParameterObject.js

```diff
@@ -106,11 +106,33 @@ function parseParameterObject(context, object) {
     [R.T, createInvalidMemberWarning(namespace, name)],
   ]);
 
-  const isPathParameter = parameter => parameter.getValue('in') === 'path';
+  const createUnsupportedNameError = createError(namespace, `'${name}' 'name' contains unsupported characters. Only alphanumeric characters are currently supported`);
+
+  const createReservedHeaderNamesWarning = createWarning(namespace, `'${name}' 'name' in location 'header' ignore keywords 'Accept', 'Content-Type' and 'Authorization'`);
```

Comment:
I think we could make this warning message a bit clearer:

```suggestion
  const createReservedHeaderNamesWarning = createWarning(namespace, `'${name}' 'name' in location 'header' should not be 'Accept', 'Content-Type' or 'Authorization'`);
```